doc/source/train/distributed-pytorch/migration-guides.rst

```diff
@@ -4,9 +4,14 @@ Migration Guides for LightningTrainer, TransformersTrainer, and AccelerateTraine
 =================================================================================
 
 In Ray 2.7, we're excited to introduce a unified approach for Torch-based training frameworks
-using the :class:`~ray.train.torch.TorchTrainer` API. This change allows for a more 
-flexible and complementary integration with PyTorch Lightning, HuggingFace Transformers, 
-and HuggingFace Accelerate.
+using the :class:`~ray.train.torch.TorchTrainer` API. 
```

Comment:
Just FYI, I am planning to move these different framework migration guides to their individual pages.